libs/api/domains/social-insurance/src/lib/models/general/union.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
```

Comment:
Remove unused import 
```suggestion
import { Field, ObjectType } from '@nestjs/graphql'
```